pkg/serverless/trace/inferredspan/constants.go

```diff
@@ -11,6 +11,7 @@ const (
 	apiName          = "apiname"
 	bucketARN        = "bucket_arn"
 	bucketName       = "bucketname"
+	busName          = "BusName"
```

Comment:
This isn't camelCase like the other constants, but I was following the PascalCase format of `SentTimestamp`.